jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/Constants.java

```diff
@@ -11,12 +11,26 @@
   private Constants() {}
 
   public static final String ONE = "1";
-  public static final String KILOBYTES = "KB";
+  public static final String KILOBYTES = "kb";
   public static final String MILLISECONDS = "ms";
   public static final String PERCENTAGE = "%age";
   public static final String READ = "read";
   public static final String WRITE = "write";
-  public static final AttributeKey<String> ATTR_THREAD_NAME = AttributeKey.stringKey("thread.name");
+  public static final String USER = "user";
+  public static final String SYSTEM = "system";
+  public static final String MACHINE = "machine_total";
+  public static final String G1 = "g1";
+  public static final String USED = "used";
+  public static final String COMMITTED = "committed";
+
+  public static final AttributeKey<String> ATTR_THREAD_NAME = AttributeKey.stringKey("thread_name");
```

Comment:
In otel, we tend to use `.` for separators, even in attributes.  See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/span-general.md#general-thread-attributes